packages/framework/src/Model/Advert/AdvertFacade.php

```diff
@@ -69,6 +77,7 @@ public function getById($advertId)
      */
     public function findRandomAdvertByPositionOnCurrentDomain($positionName)
     {
+        $this->advertPositionRegistry->assertPositionNameIsKnown($positionName);
```

Comment:
We usually separate return statement with a blank line. In the past, this was (still is?) a part of the coding standards.